tensorly/backend/core.py

```diff
@@ -574,6 +574,17 @@ def conj(x, *args, **kwargs):
         """
         raise NotImplementedError
 
+    @staticmethod
+    def values(tensor):
+        """ Returns all the non zero values of the tensor in COO
+        """
+        raise NotImplementedError
+
+    @staticmethod
+    def indices(tensor):
+        """ Returns all the indices of non zero values of the tensor in COO
+        """
+        raise NotImplementedError
 
```

Comment:
I think in order to have static `indices` and `values` methods for all sparse backends, I had to provide an implementation for the non-sparse backends too, which is why I am changing `tensorly/backend/core.py`